ngraph_onnx/onnx_importer/backend.py

```diff
@@ -118,8 +118,8 @@ def run_model(cls, onnx_model, inputs, device='CPU', **kwargs):
         return cls.prepare(onnx_model, device, **kwargs).run(inputs)
 
     @classmethod
-    def run_node(cls, onnx_node, inputs, device='CPU'):
-        # type: (onnx.NodeProto, List[np.ndarray], str) -> List[np.ndarray]
+    def run_node(cls, onnx_node, inputs, opset=4, device='CPU'):
```

Comment:
Use `opset_version` instead of `opset`. Don't set the default value here, let's stick to ONNX API